en/mkdocs.yml

```diff
@@ -178,87 +178,89 @@ nav:
-            - HTTP 2.0: deploy/api-microgateway/http-2.0.md
-            - Rate Limiting:
-                 - Enabling distributed throttling: deploy/api-microgateway/rate-limiting/distributed-throttling.md
-            - Service Discovery: deploy/api-microgateway/service-discovery.md
-            - Security:
-                - Authentication:
-                    - API Authentication: deploy/api-microgateway/security/api-authentication/api-authentication.md
-                    - Secure APIs Using OAuth2.0 Access Tokens:
-                        - Secure APIs using JWT - Self Contained JWT: deploy/api-microgateway/security/api-authentication/secure-apis-using-oauth2.0-access-tokens/secure-apis-using-jwt-self-contained-jwt.md
-                        - Secure APIs Using Opaque Tokens: deploy/security/api-authentication/secure-apis-using-oauth2.0-access-tokens/secure-apis-using-opaque-tokens.md
-                    - Mutual SSL Authentication: deploy/security/api-authentication/mutual-ssl-authentication.md
-                    - Configuring an External Key Manager: deploy/security/api-authentication/configuring-an-external-key-manager.md
-                    - Making Application Security Optional: deploy/security/api-authentication/making-application-security-optional.md
-                    - Disabling Security: deploy/security/api-authentication/disabling-security.md
-                - Authorization:
-                    - Scope Validation: deploy/api-microgateway/security/api-authorization/scope-validation.md
-                    - Subscription Validation: deploy/security/api-authorization/subscription-validation.md
-                - Transport Security: deploy/security/transport-security.md
-                - Enabling CORS: deploy/security/enabling-cors.md
-                - Use a Custom Authorization Header: deploy/security/use-a-custom-authorization-header.md
-                - Rejecting Revoked Tokens: deploy/security/rejecting-revoked-tokens.md
-                - Importing Certificates to the API Microgateway Truststore: deploy/security/importing-certificates-to-the-api-microgateway-truststore.md
-                - Define Security Using Extension: deploy/security/define-security-using-extension.md
-            - File Upload via Multipart Requests: deploy/file-upload-via-multipart-requests.md
-            - Configure Caching: deploy/configure-caching.md
-            - Manage Microgateway APIs with WSO2 API Manager: deploy/manage-microgateway-apis-with-wso2-api-manager.md
-            - Passing Enduser Attributes to the Backend Using JWT: deploy/passing-enduser-attributes-to-the-backend-using-jwt.md
-            - Custom Claims Mapping: deploy/support-custom-claims-mapping.md
+    - Deploy and Publish:
+        - Deploy on Gateway:
+            - Deploying APIs in API Gateway vs API Microgateway: deploy-and-publish/deploy-on-gateway/deploying-apis-in-api-gateway-vs-api-microgateway.md
```

Comment:
We need some kind of section overview. This was it previously. Can we reuse the content in this somehow or create a new one?

Reply:
This is still there. Did not remove it.

Reply:
Right now it is the overview for the Deploy on Gateway section. We need one for the overall section as well. If we are using the same thing, it has to change a bit and come out of the Gateway section.